### ISSUE TITLE
Add PCCW Parser

### ISSUE THREAD

**Issue by ryanmerolle** [networktocode/circuit-maintenance-parser#173]
### Environment

- circuit_maintenance_parser version: 2.1.0

### Proposed Functionality

Add PCCW maintenance parser

### Use Case

Useful for clients PCCW.

**Comment by chadell** [comment]
Thanks @ryanmerolle 
By any chance, do you have an example of a notification from this provider?

**Comment by ryanmerolle** [comment]
Of course.  Do you need the unaltered email, or can an anonymized PDF work for now?

I did not see any guidelines on submitting the examples.

**Comment by chadell** [comment]
There is no guideline to submit examples as usually they come with the parser. You could attach them here, anonymised (but with the original format)

**Comment by ryanmerolle** [comment]
I will do so shortly.  Thanks!

**Comment by snitass** [comment]
I would like to add PCCW parser as well. Would you mind if I took over this?

**Comment by snitass** [comment]
Their urgent, planned, cancellation and amendment notifications are coming with ics, so I will use SimpleProcessor. The completion notification doesn't come with ics, hence; I will create custom parser for it. Please let me know if I misunderstood anything here.

Thanks

**Comment by chadell** [comment]
yes, the idea is to combine different Parsers into one Processor that gets the data from where available.

**Comment by snitass** [comment]
https://github.com/networktocode/circuit-maintenance-parser/pull/301
